src/terra-dev-site/app/components.2/NotificationBanner.app.mdx

```diff
@@ -0,0 +1,74 @@
+import PropsTable from './NotificationBannerPropsTable?dev-site-props-table';
+import NotificationBannerExample from './example/NotificationBannerExample?dev-site-example';
+
+# Notification Banner
+
+A Notification Banner should be rendered by a component that needs to render a notification with information that needs to be brought to the user's attention. This is a non-disruptive notification that does not require immediate input or attention by the user.
+
+When rendered, the Notification Banner registers itself with the framework so it prioritized correctly when any other rendered notification banners before it is displayed in a list above all other content.
+
+## Banner Variants
+
+The Notification Banner can be used to bring awarness to the a user’s risk and non-risk situations that may occur within an application; there are multiple variants provided for common situtations. 
+
+- Notification Banners show for risk situations are intended to notify the user of hazards, such as risks to patient safety or causing financial harm, and provide options to mitigate or remove the hazard. There are three variants of risk oriented notification banners, based on the severity: `hazard-high`, `hazard-medium` and `hazard-low`. These variants are rigid to provide consistent icon and signal word pairs to prevent accidental confusion by the user of the banner's severity.
```

Comment:
```suggestion
- Notification Banners shown for risk situations are intended to notify the user of hazards, such as risks to patient safety or causing financial harm, and provide options to mitigate or remove the hazard. There are three variants of risk oriented notification banners, based on the severity: `hazard-high`, `hazard-medium` and `hazard-low`. These variants are rigid to provide consistent icon and signal word pairs to prevent accidental confusion by the user of the banner's severity.
```